hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -6674,7 +6674,7 @@ public String getLiveNodes() {
           .put("infoSecureAddr", node.getInfoSecureAddr())
           .put("xferaddr", node.getXferAddr())
           .put("location", node.getNetworkLocation())
-          .put("uuid", node.getDatanodeUuid())
+          .put("uuid", node.getDatanodeUuidForMetric())
```

Comment:
rather than creating a new method, can't we handle the null case here itself?
something like
```
.put("uuid", Optional.ofNullable(node.getDatanodeUuid()).orElse(""))
```

Reply:
Thanka @ayushtkn for your commet.
Update PR, please help review it again , thanks!